packages/discovery-react-components/src/components/SearchResults/SearchResults.tsx

```diff
@@ -256,6 +273,45 @@ const SearchResults: React.FunctionComponent<SearchResultsProps> = ({
   );
 };
 
+/**
+ * Hook to update search parameters' `return` param with the minimal necessary fields
+ * that are necessary to render this component.
```

Comment:
Confused a bit by the wording of this comment, before I go read the actual code. Is this closer to a reset-to-defaults hook? Or are there some parts of the search parameters that will remain the same, meanwhile others are getting trimmed off?